packages/pymedphys_gamma/src/pymedphys_gamma/interpolate/__init__.py

```diff
@@ -0,0 +1,4 @@
+try:
+    from scipy.interpolate import RegularGridInterpolator
+except ImportError:
+    from .regulargrid import RegularGridInterpolator
```

Comment:
Tries to import from scipy first...

Reply:
Removed this in the end... figure it might just get confusing if it can import from two places. Keep it just to one then a failed/passed test is indicative of reality everywhere